[rospy.client][INFO] 2019-06-25 09:40:08,530: init_node, name[/rostopic_27209_1561455608417], pid[27209]
[xmlrpc][INFO] 2019-06-25 09:40:08,532: XML-RPC server binding to 0.0.0.0:0
[rospy.init][INFO] 2019-06-25 09:40:08,533: ROS Slave URI: [http://dc5b429f784e:33349/]
[xmlrpc][INFO] 2019-06-25 09:40:08,533: Started XML-RPC server [http://dc5b429f784e:33349/]
[rospy.impl.masterslave][INFO] 2019-06-25 09:40:08,534: _ready: http://dc5b429f784e:33349/
[rospy.registration][INFO] 2019-06-25 09:40:08,534: Registering with master node http://localhost:11311
[xmlrpc][INFO] 2019-06-25 09:40:08,535: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2019-06-25 09:40:08,634: registered with master
[rospy.rosout][INFO] 2019-06-25 09:40:08,634: initializing /rosout core topic
[rospy.rosout][INFO] 2019-06-25 09:40:08,641: connected to core topic /rosout
[rospy.simtime][INFO] 2019-06-25 09:40:08,644: initializing /clock core topic
[rospy.simtime][INFO] 2019-06-25 09:40:08,649: connected to core topic /clock
[rospy.internal][INFO] 2019-06-25 09:40:08,652: topic[/clock] adding connection to [http://dc5b429f784e:33033/], count 0
[rospy.internal][INFO] 2019-06-25 09:40:08,767: topic[/odom] adding connection to [http://dc5b429f784e:33033/], count 0
[rospy.internal][INFO] 2019-06-25 09:40:08,930: topic[/rosout] adding connection to [/rosout], count 0
[rospy.core][INFO] 2019-06-25 09:40:09,594: signal_shutdown [signal-2]
[rospy.internal][INFO] 2019-06-25 09:40:09,601: topic[/rosout] removing connection to /rosout
[rospy.internal][ERROR] 2019-06-25 09:40:09,601: Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 729, in receive_once
    self.stat_bytes += recv_buff(sock, b, p.buff_size)
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 103, in recv_buff
    d = sock.recv(buff_size)
error: [Errno 104] Connection reset by peer

[rospy.internal][INFO] 2019-06-25 09:40:09,602: topic[/clock] removing connection to http://dc5b429f784e:33033/
[rospy.internal][INFO] 2019-06-25 09:40:09,602: topic[/odom] removing connection to http://dc5b429f784e:33033/
[rospy.impl.masterslave][INFO] 2019-06-25 09:40:09,602: signal-2
[rospy.internal][WARNING] 2019-06-25 09:40:09,602: Unknown error initiating TCP/IP socket to dc5b429f784e:58633 (http://dc5b429f784e:33033/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 558, in connect
    self.local_endpoint = self.socket.getsockname()
AttributeError: 'NoneType' object has no attribute 'getsockname'

[rospy.core][INFO] 2019-06-25 09:40:09,735: signal_shutdown [atexit]
